Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/qrcode/LocationQRCodeVerifier.kt

```diff
@@ -2,15 +2,14 @@ package de.rki.coronawarnapp.eventregistration.checkins.qrcode
 
 import de.rki.coronawarnapp.server.protocols.internal.pt.TraceLocationOuterClass
 import de.rki.coronawarnapp.util.security.SignatureValidation
-import okio.ByteString.Companion.decodeBase64
 import timber.log.Timber
 import javax.inject.Inject
 
-class DefaultQRCodeVerifier @Inject constructor(
+class LocationQRCodeVerifier @Inject constructor(
```

Comment:
I would re-use this Verifier to verify the traceloation that is returned from the server. Does it make sense to rename this class to something like `TraceLocationVerifier`?

Reply:
Wait... I don't need to use this class for my task. I can simply use `SignatureValidation`, so the comment above is obsolete.